examples/smartContracts/deployer.ts

```diff
@@ -79,7 +79,12 @@ export const deploySmartContracts = async (
   try {
     // do checks
     if (!deployerAccount) {
-      deployerAccount = web3Client.wallet().getBaseAccount()!;
+      const baseAccount = web3Client.wallet().getBaseAccount();
+      if (baseAccount === null) {
```

Comment:
Maybe a ternary for if/else pattern to simplify, or even better since it throws:
an assertion ?
e.g:
```ts
deployerAccount = web3Client.wallet().getBaseAccount()
assert(deployerAccount !== null, "Failed to use base account");
```

Reply:
Oh yeah you are right, forgot how dumb eslint was, you can mark as resolved then.